Frends.AzureBlobStorage.ListBlobsInContainer/Frends.AzureBlobStorage.ListBlobsInContainer/Definitions/Source.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+namespace Frends.AzureBlobStorage.ListBlobsInContainer.Definitions
+{
+    /// <summary>
+    ///     Source values.
+    /// </summary>
+    public class Source
+    {
+        /// <summary>
+        ///     Authentication method to use when connecting to Azure Storage. Options are connection string and SAS Token.
```

Comment:
Could we have an explicit default value for this? This would help both documentation and some ambiguousness when reading code.